pkg/manifests/manifests.go

```diff
@@ -189,17 +233,30 @@ func newIngressControllerClusterRoleBinding(conf *config.Config) *rbacv1.Cluster
 			Name:     IngressControllerName,
 		},
 		Subjects: []rbacv1.Subject{{
-			Kind:      "ServiceAccount",
+			Kind:      serviceAccountKind,
 			Name:      IngressControllerName,
 			Namespace: conf.NS,
 		}},
 	}
 }
 
+func getIngressControllerService(conf *config.Config) *corev1.Service {
```

Comment:
Would prefer to merge this logic into `newIngressControllerService` to avoid adding another "hop" for future readers